CorsixTH/Lua/dialogs/fullscreen/town_map.lua

```diff
@@ -188,7 +188,8 @@ function UITownMap:draw(canvas, x, y)
   self.info_font:draw(canvas, radiators,    x +  95, y + 265)
 
   -- Heating costs
-  local heating_costs = math.floor(((hospital.heating.radiator_heat *10)* radiators)* 7.5)
+  local heating_costs = app.config.free_build_mode and 0
+      or math.floor(((hospital.heating.radiator_heat *10)* radiators)* 7.5)
```

Comment:
Move the `or` up a line

Reply:
Can we do this for him; I hate to discourage a new dev too much with nits.